apidoc/Titanium/UI/Switch.yml

```diff
@@ -35,12 +35,10 @@ methods:
         to appear on top of it.
 
         See also: <Titanium.UI.View.add>.
-    platforms: []
 
   - name: remove
     description: |
         See also: [add](Titanium.UI.Switch.add), <Titanium.UI.View.add>.
-    platforms: []
```

Comment:
I don't think these methods should stay. Instead, it should be added to the `exclude` array of inherited methods.

Reply:
I'm actually okay with this because you can't add/remove child views to a Switch on Android either.

Reply:
So now the docs will indicate that it works for both iOS and Android because thats the behavior when there is no `platforms` tag. Not sure if I like that.

Reply:
`Switch` inherits from `View`. The `View` type already has `add()` and `remove()` methods. What the above documentation indicates is that these methods are not supported on all platforms, which is true. You can call these methods on Android, but they'll no-op.

Reply:
What I meant above is to do it like [here](https://github.com/sgtcoolguy/titanium_mobile/blob/63d1c4afe2288763d391264174443e890b475520/apidoc/Titanium/UI/AlertDialog.yml#L75), which couldn't be done in the past because Mobileweb did support it. Let's just leave it then.